protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -4625,10 +4625,13 @@ message GetTotalTransferFeeDiscountResponse {
 message ListGamesRequest {
   // Game ID to filter for.
   optional string game_id = 1;
-  // First epoch to retrieve game data from. If not provided, defaults to the last epoch minus 30,
-  // or the first epoch available if not enough epochs have passed.
+  // First epoch to retrieve game data from (inclusive). If not provided, defaults to the last epoch minus 29,
+  // or the first epoch available if not enough epochs have passed. The maximum range of epochs returned is 30.
+  // If the last epoch is not specified, it will be set to first epoch + 29.
```

Comment:
```suggestion
  // Oldest epoch to retrieve game data from, inclusive. The maximum range of epochs that can be fetched at once is 30. If not provided, defaults to the 'to' epoch minus 29,
  // or the first epoch available if not enough epochs have passed.
  // If the 'to' epoch is not specified, it will be set to the chosen 'from' epoch + 29.
```